.github/workflows/tests.yml

```diff
@@ -101,6 +101,19 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
+  rs-features:
```

Comment:
Let's just add the job form ./.github/workflows/rs-features.yml to tests-rs-package workflow